source/CommonJobs/CommonJobs.Application.EvalForm/Helper/AlphabeticallyStringComaprer.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CommonJobs.Application.EvalForm.Helper
+{
+    public class AlphabeticallyStringComaprer : IComparer<string>
```

Comment:
There should be something already existent in the framework to avoid to create it from the scratch.